pages/1_🎵_楽曲検索.py

```
@@ -132,24 +127,37 @@ def get_recently_added_songs(
-        song_list.append((song_id, metadata))
-
-    # registered_atでソート（新しい順）
-    # registered_atが存在しない場合は古い扱いとする
-    sorted_songs = sorted(
-        song_list,
-        key=lambda x: x[1].get("registered_at", "1900-01-01T00:00:00"),
-        reverse=True,  # 新しい順
-    )
-
-    return sorted_songs[:limit]
+    # MySQLから最近追加された曲を取得（ORDER BY registered_at DESC）
```

Ruff (0.14.14)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)